aptos-move/framework/src/natives/transaction_context.rs

```diff
@@ -387,7 +406,10 @@ pub fn make_all(
     let natives = [
         ("get_script_hash", native_get_script_hash as RawSafeNative),
         ("generate_unique_address", native_generate_unique_address),
-        ("get_txn_hash", native_get_txn_hash),
+        (
```

Comment:
just add both with same:

```
("get_txn_hash", native_unique_session_hash_internal),
("unique_session_hash_internal", native_unique_session_hash_internal),
```

Reply:
I added it back. Now, we have both.

Reply:
@igor-aptos , since `get_transaction_hash` is still available, should `get_transaction_hash` still be used in this release instead of `unique_session_hash`?